refactor: Cleaned up the codebase

BlazeMCworld opened the pull request:
- Formatted code
- Updated deprecated code
- Removed unused imports

BlazeMCworld committed:
refactor: Cleaned up the codebase
- Formatted code
- Updated deprecated code
- Removed unused imports

Chickensoup20 commented:
😍

RedVortexDev commented on src/main/java/hypersquare/hypersquare/command/DeleteAllPlotsCommand.java:
```diff
@@ -23,17 +25,18 @@
+                Utilities.sendInfo(player, "Dumped all plots");
+                for (Player player1 : Bukkit.getOnlinePlayers()) {
+                    player1.kick(Component.text("Kick from Hypersquare").color(NamedTextColor.RED).decoration(TextDecoration.BOLD, true)
+                            .append(Component.text("Reason: massive dump").color(NamedTextColor.RED))
```
Not the same as a \n

RedVortexDev commented on src/main/java/hypersquare/hypersquare/command/LocateCommand.java:
```diff
@@ -27,8 +28,12 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                 target = player;
             }
+            if (target == null) {
+                player.sendMessage(Component.text("Player not found.").color(NamedTextColor.RED));
```
Should be replaced with Utilities.sendError()

RedVortexDev commented on src/main/java/hypersquare/hypersquare/plot/CodeBlockManagement.java:
```diff
@@ -49,62 +41,39 @@ public static void placeBracket(Location location, String codeblock, String brac
-    }
-
-    public static Location findNextBracket(Location location){
-        while (location.clone().add(0,0,1).getBlock().getType() != Material.AIR || location.clone().add(0,0,2).getBlock().getType() != Material.AIR || location.clone().getBlock().getType() != Material.AIR){
```
why were this deleted?

BlazeMCworld committed:
refactor: pr feedback
- Add a newline to the kick msg
- Use Utilities.sendError()
- Restore findNextBracket

RedVortexDev reviewed (approved):
LGTM
